x-pack/plugins/stack_connectors/public/connector_types/bedrock/params.tsx

```diff
@@ -67,7 +67,7 @@ const BedrockParamsFields: React.FunctionComponent<ActionParamsProps<BedrockActi
         paramsProperty={'body'}
         inputTargetValue={body}
         label={i18n.BODY}
-        aria-label={i18n.BODY_DESCRIPTION}
+        areaLabel={i18n.BODY_DESCRIPTION}
```

Comment:
maybe we should fix this typo in the props of `JsonEditorWithMessageVariables` :)

Reply:
Good point, updated via d06e641010d2ab105e46e96ca75f037cf6ed4bd2